doc/reference_coverage.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sect1 id="Coverage">
+  <sect1info>
+    <abstract>
+      <para>These functions operate on collections of polygonal geometry that form "implicit coverages" in which any edges shared by the polygons match exactly. These implicit coverages are fast to compute, and can be operated on with window functions, which retain the implicit topology inside the window partition while altering the edges.</para>
```

Comment:
I'm not sure what "These implicit coverages are fast to compute" means. What is being computed?

Reply:
Essentially all polygonal coverage operations are faster to compute if the coverage topology is taken into account.  

Perhaps better to say "fast to process".   The idea is to alert users to the benefit of modelling data as a coverage, and using coverage operations.